concent-builder-vm/readme.txt

```diff
@@ -0,0 +1,47 @@
+# HOW TO SETUP A VAGRANT MACHINE
```

Comment:
The instructions for installing Vagrant, VirtualBox, searching for a box on the web page, downloading it, etc. are not needed.

Installing dependencies is just a matter of installing the `vagrant` package and instructions for doing that are system dependent. We can safely assume that a competent developer will know how to install a package in his own system. And if not, it's not in the scope of our project scope to try to explain it.

Vagrant box is already hard-coded in your `Vagrantfile` so no need to explain the search process either. The box will be downloaded automatically on the first run. It's all a matter of a single instruction - `vagrant up`.

So, in summary, it's enough to just mention in the main README file that `build-server-vm` contains configuration script for a virtual machine, that it depends on Vagrant and that you create it by running `vagrant up`. That's it. Please remove the commit with these instructions. You can put them on Github wiki if you really want to keep it.